README.md

```diff
@@ -97,3 +97,26 @@ If you need help with this plugin, please file an issue explaining the following
 - Why you think this is wrong
 
 Issues that are not filed with this information will be closed. We will do our best to assist, but we cannot guarantee a response.
+
+## CHANGELOG
+
+0.6 - Nov 13 2024
```

Comment:
Note for our future selves: Be sure to update this on release-day 🙂

Reply:
I've found having a separate file for the changelog works much better. I'm not sure how AP will handle this or is the intent to copy how .org parses the readme.txt file.

If so, coincidentally there's a repository at `afragen/wordpress-plugin-readme-parser` that is available on packagist. It is an up to date version of .org's parser and is used by the [Plugin Check plugin](https://github.com/WordPress/plugin-check/blob/608225ab2507ac393d94c3980f132c8283b9dc1a/composer.json#L9).

Reply:
Yeah we will get there when we use GA to do a separate changelog file.